Source/Core/CMakeLists.txt

```diff
@@ -19,3 +19,8 @@ if (APPLE)
   add_subdirectory(UpdaterCommon)
   add_subdirectory(MacUpdater)
 endif()
+
+if (WIN32)
+  add_subdirectory(UpdaterCommon)
```

Comment:
Any reason why UpdaterCommon can't be added unconditionally?

Reply:
We don't need it on every platform.